### PR TITLE
CI/TST: Skip external s3 bucket test that hangs on min build

### PR DESCRIPTION
**Pull request by mroeschke**

### CODE REVIEW

**Comment by mroeschke** [comment]
Looks like one of the processes got stuck on `test_s3_parser_consistency`

```
2022-01-27T23:11:31.0333115Z [gw1] PASSED pandas/tests/io/xml/test_xml.py::test_unsuported_compression[etree] 
2022-01-27T23:11:35.6861543Z pandas/tests/io/xml/test_xml.py::test_s3_parser_consistency 
2022-01-27T23:11:35.6944349Z [gw0] PASSED pandas/tests/io/sas/test_sas7bdat.py::TestSAS7BDAT::test_iterator_loop 
```

In https://github.com/pandas-dev/pandas/pull/45665/commits/1fd4e923accf92bed8bedd4dab274f2e3d36a592, I don't see that test ever complete in the logs.

**Comment by mroeschke** [comment]
@jbrockmendel min build is green here. Wdyt?

File: pandas/tests/io/xml/test_xml.py [hunk 1]
```diff
@@ -1102,6 +1102,10 @@ def test_unsuported_compression(datapath, parser):
 @tm.network
 @td.skip_if_no("s3fs")
 @td.skip_if_no("lxml")
+@pytest.mark.skipif(
+    os.environ.get("PANDAS_CI", "0") == "1",
+    reason="2022.1.17: Hanging on the CI min versions build.",
+)
```
Comment (jbrockmendel) [inline pandas/tests/io/xml/test_xml.py:1108]: You’re confident this is the culprit?

File: ci/run_tests.sh [hunk 2]
```diff
@@ -24,7 +24,7 @@ if [[ $(uname) == "Linux" && -z $DISPLAY ]]; then
     XVFB="xvfb-run "
 fi
 
-PYTEST_CMD="${XVFB}pytest -r fEs -m \"$PATTERN\" -n $PYTEST_WORKERS --dist=loadfile $TEST_ARGS $COVERAGE $PYTEST_TARGET"
+PYTEST_CMD="${XVFB}pytest -v -r fEs -m \"$PATTERN\" -n $PYTEST_WORKERS --dist=loadfile $TEST_ARGS $COVERAGE $PYTEST_TARGET"
```
Comment (jbrockmendel) [inline ci/run_tests.sh:27]: Do we want to keep this or was it just for troubleshooting?

Comment (mroeschke) [reply ci/run_tests.sh:27]: Good catch

File: pandas/tests/io/xml/test_xml.py (hunk 1 above)
Comment (mroeschke) [reply pandas/tests/io/xml/test_xml.py:1108]: Fairly. I'll run one more time removing the `-v`

Comment (jbrockmendel) [reply pandas/tests/io/xml/test_xml.py:1108]: Sounds good to me.

**Comment by mroeschke** [comment]
All green on the min build. Going to merge.

**Pull request merged**